projects/lib-project/src/lib/dock/components/certificates/certificates.component.scss

```diff
@@ -0,0 +1,41 @@
+@import  "../../../../../../self-creation-portal/src/styles.scss";
+.required::after {
+    content: '*';
+    color: $warn;
+    font-size: x-large;
+}
+
+input:focus{
+    outline: none;
+}
+
+.disabled-class {
+    cursor:not-allowed;
+    pointer-events: none;
+    opacity: 0.5; 
+    color:$buttonDisable;
+}
+
+.error-message {
```

Comment:
isn't this available in styles.scss file ?

Reply:
this is not available in style.scss